playwright/_impl/_browser_type.py

```diff
@@ -135,6 +136,26 @@ async def launch_persistent_context(
                 raise not_installed_error(f'"{self.name}" browser was not found.')
             raise e
 
+    async def connect(
+        self, ws_endpoint: str, timeout: float = 30000, slow_mo: float = None
+    ) -> Browser:
+        transport = WebSocketTransport(ws_endpoint, timeout)
+        from playwright._impl._object_factory import create_remote_object
```

Comment:
import at top

Reply:
The reason for that was to quickly workaround the circular dependency. But I ended up now locally to use the object_factory from the parent connection. Seems also better.